src/evict/evict_lru.c

```diff
@@ -1977,8 +1977,10 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
         }
 
         /* Don't queue dirty pages in trees during checkpoints. */
-        if (modified && WT_BTREE_SYNCING(btree))
+        if (modified && WT_BTREE_SYNCING(btree)) {
+            WT_STAT_CONN_INCR(session, cache_eviction_skip_queuing_dirty_pages_during_checkpoint);
```

Comment:
nit: We haven't used "queuing" in the other stats. Can we remove it here also?
`cache_eviction_skip_dirty_pages_during_checkpoint`

Reply:
Fixed in f6176b656d9a0112686d210cdf8d28783dc815e8